KPCTabsControl/DefaultTheme.swift

```diff
@@ -26,15 +26,15 @@ public struct DefaultTheme: Theme {
     private static var sharedBackgroundColor: NSColor { return NSColor(calibratedWhite: 0.95, alpha: 1.0) }
     private static var sharedHighlightedBackgroundColor: NSColor { return NSColor(calibratedWhite: 0.85, alpha: 1.0) }
 
-    struct DefaultTabButtonTheme: KPCTabsControl.TabButtonTheme {
+    private struct DefaultTabButtonTheme: KPCTabsControl.TabButtonTheme {
         
         var backgroundColor: NSColor { return DefaultTheme.sharedBackgroundColor }
         var borderColor: NSColor { return DefaultTheme.sharedBorderColor }
         var titleColor: NSColor { return NSColor.darkGrayColor() }
         var titleFont: NSFont { return NSFont.systemFontOfSize(13) }
     }
 
-    struct HighlightedTabButtonTheme: KPCTabsControl.TabButtonTheme {
+    private struct HighlightedTabButtonTheme: KPCTabsControl.TabButtonTheme {
```

Comment:
It's already commited and pushed yesterday evening normally. I'll check on my side but... have a look in yours. :-)
Cédric

Sent from an iPhone 6s running iOS9 at 8 kiloparsecs from the Galactic Center

> Le 18 août 2016 à 10:04, Christian Tietze <notifications@github.com> a écrit :
> 
> In KPCTabsControl/DefaultTheme.swift:
> 
> >          
> >          var backgroundColor: NSColor { return DefaultTheme.sharedBackgroundColor }
> >          var borderColor: NSColor { return DefaultTheme.sharedBorderColor }
> >          var titleColor: NSColor { return NSColor.darkGrayColor() }
> >          var titleFont: NSFont { return NSFont.systemFontOfSize(13) }
> >      }
> >  
> > -    struct HighlightedTabButtonTheme: KPCTabsControl.TabButtonTheme {
> > +    private struct HighlightedTabButtonTheme: KPCTabsControl.TabButtonTheme {
> can we get rid of this variant completely, since there's no highlighting anymore?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>